command/runner.go

```diff
@@ -18,20 +22,30 @@ type Args struct {
 	Stdin       pulumi.StringPtrInput
 	Environment pulumi.StringMap
 	Sudo        bool
+	User        string
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Wonder if there is a way to wrap a command inside a `sudo` command, with `sudo` options, instead of passing `sudo` and `user` to args. `User` makes sense only in `sudo` context, it is not ideal to have two options changing the same behaviour - it can be confusing, what if I pass `sudo = false` and `user: "something` ?

Reply:
One option is to remove `Sudo` and keep `User`. In order to run commands as root the `User` must be set to `root`. The prefix will then be `sudo -u root`. I am not sure if this is intuitive though...?

Reply:
Alternatively, `Sudo` can be renamed to `Root` or `AsRoot`, and the behavior can be to override `User` and run the command as root.

Reply:
I will revert to the old way for now, and address this in a different PR.